url.bs

```diff
@@ -3345,20 +3345,19 @@ console.log(url.searchParams.get('b')); // "~"</code></pre>
 
 <ol>
  <li>
-  <p>If <var>init</var> is a <a>sequence</a>, then <a for=list>for each</a> <var>pair</var> in
+  <p>If <var>init</var> is a <a>sequence</a>, then <a for=list>for each</a> <var>tuple</var> of
```

Comment:
The variable name could still be "pair".

Reply:
While true, I think I prefer sticking to a single term throughout. I think that makes it easier for non-native speakers.

Reply:
Honestly I think _subSequence_ or _innerSequence_ would be a better name. It's confusing to use the variable name _tuple_ for something that's actually an Infra list/IDL sequence.

For example, at first I wrote a comment below "Why not just append _tuple_ directly, instead of appending (_tuple_[0], _tuple_[1])?" But now I see that would be a type error. I was misled by the variable name.